eth_defi/gmx/core/borrow_apr.py

```diff
@@ -0,0 +1,75 @@
+"""
+GMX Borrow APR Data Retrieval Module.
+
+This module provides borrow APR data for GMX protocol markets.
+"""
+
+import logging
+from typing import Any
+
+from eth_defi.gmx.config import GMXConfig
+from eth_defi.gmx.core.get_data import GetData
+from eth_defi.gmx.types import PositionSideData
+
+
+class GetBorrowAPR(GetData):
+    """GMX borrow APR data retrieval class.
+
+    Retrieves borrow APR information for all available GMX markets,
+    replacing the gmx_python_sdk GetBorrowAPR functionality.
+    """
+
+    def __init__(self, config: GMXConfig, filter_swap_markets: bool = True):
+        """Initialize borrow APR data retrieval.
+
+        :param config: GMXConfig instance containing chain and network info
+        :param filter_swap_markets: Whether to filter out swap markets from results
+        """
+        super().__init__(config, filter_swap_markets)
+        self.log = logging.getLogger(__name__)
```

Comment:
Instead of `self.log` use the standard `logger = logging.getLogger(__name__)` module pattern that all other modules in `eth_defi` use.